### DISCUSSION TITLE
Custom themes

### DISCUSSION THREAD

**Discussion by sillypnk** [mantinedev/mantine#1472] [Q&A] [answered]
Hello, is it possible to make custom themes with mantine besides of dark or light?

**Comment by rtivital** [comment, marked as answer]
See #1300

**Reply by sillypnk** [reply]
Thanks! btw can we have multiple themes?

**Reply by rtivital** [reply]
theme is a javascript object, you can create as many as you need

**Reply by sillypnk** [reply]
> theme is a javascript object, you can create as many as you need

but somehow the colors are not same as the colors i want like [daisyui's](https://daisyui.com/docs/themes/) themes
is the're anyway to change the exact colors?

![brave_OrBLuKmkq6](https://user-images.githubusercontent.com/102006493/168948162-b4464a09-f2df-4af4-b85c-36ad13974bc5.png)


![brave_XBgp4KccMM](https://user-images.githubusercontent.com/102006493/168948116-82a7ee33-6105-455a-b088-5087a4cc246e.png)

**Reply by rtivital** [reply]
You can change colors in `theme.colors` as you like – you define these values on your side
